test/instrumentation/github-issue-2715.test.js

```diff
@@ -0,0 +1,52 @@
+const agent = require('../..').start({
+  serviceName: 'test-composite-context',
+  captureExceptions: false,
+  metricsInterval: 0,
+  centralConfig: false,
+  cloudProvider: 'none',
+  spanCompressionEnabled: true
```

Comment:
nit: The `APM Server transport error: error fetching APM Server version:` test output error (shown below) could be avoided by setting `apmServerVersion: '8.0.0',` or similar in the agent config here.